source/_integrations/shelly.markdown

```diff
@@ -127,6 +127,14 @@ The integration uses the following strategy to name its entities:
 - If `Channel Name` is set in the device, the integration will use it to generate the entities' name, e.g. `Kitchen Light`
 - If `Channel Name` is set to the default value, the integration will use the `Device ID` and default channel name to generate the entities' name, e.g. `ShellyPro4PM-9808D1D8B912 switch_0`.
 
+## Cover entities
+
+Shelly 2PM Gen3 supports `tilt` for `cover` entities. To enable this feature, you need to configure the device:
+
```

Comment:
nitpick question: are these steps done in this order? or can be done in any order?

Reply:
Once you have done the first step, the remaining two can be done in any order.

Reply:
ah, ok. thx. I asked because I wondered if this should be numbered steps.